### ISSUE TITLE
Glitchy behavior while trying to produce a WFM modulated sine-wave. 

### ISSUE THREAD

**Issue by m-anish** [myriadrf/gr-limesdr#72]
To properly explain the issue I'm facing, I recorded a video showing Oscilloscope waveforms. 

https://drive.google.com/file/d/11anGShu-I3NhL9Jet6YAiyBprvwP4kDc/view?usp=sharing

The gnuradio flowgraph is here:
https://drive.google.com/file/d/1xQz1Kp_feAO1YrZRXXnMF25XEGgqVPSw/view?usp=sharing

The various versions of the software/hardware involved are the following.

gnuradio-companion: 3.8.2.0 (Python 3.6.9)
gr-limesdr: branch gr-3.8 (last commit 47511dd58de1695b70e1028366411bada85eb60f)
sdrangel: 4.12.1
OS: Linux Mint 19.1
CPU: Intel© Core™ i7-4900MQ CPU @ 2.80GHz × 4
RAM: 16GB
H/w: Thinkpad T440p

tl;dr
I'm trying a simple test to create a sinewave output from a WFM modulator block. If there is no audio input, its WFM modulated signal should be a simple sinewave. If I test this with SDRAngel, I see a clean-ish actual sinewave in time domain on the oscilloscope. But if I try to do the same with gnuradio, it seems to produce a glitchy signal.

Could I have any advice on what I might be doing wrong?

Thanks,
Anish // VU2TVE

**Comment by m-anish** [comment]
https://lists.gnu.org/archive/html/discuss-gnuradio/2021-03/msg00081.html

Discussion thread on gnuradio-discuss

**Comment by m-anish** [comment]
Some more tests
https://discourse.myriadrf.org/t/glitchy-behavior-gr-limesdr-limesdr-usb-sinewave-test/7022/10?u=vu2tve
